### PR TITLE
feat(ci): send emails to slack on e2e failure

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

**Comment by Joshua-Anderson** [comment]
This is ready for reviews! [Here](https://deis.slack.com/files/slackbot/F1T8C7SPN/workflow_cli_e2e_test_failure) is an example message.

File: settings/settings.go [hunk 1]
```diff
@@ -59,13 +59,13 @@ func Load() (*Settings, error) {
...
 
-	// Set a custom user agent
-	c.UserAgent = UserAgent
-
 	if err != nil {
 		return nil, err
 	}
 
+	// Set a custom user agent
+	c.UserAgent = UserAgent
```
Comment (arschles) [inline settings/settings.go]: @Joshua-Anderson didn't you make this change in another PR (in a different repo)?

Comment (Joshua-Anderson) [inline settings/settings.go]: Sorry, that commit accidentally got caught in this PR 😦. I didn't fix it cause the PR it comes from  (#123) was about to land but forgot that it was still caught in endless CI.

Comment (arschles) [inline settings/settings.go]: got it. ok, I'm fine with both being in at once. can you make sure to add a short description regarding what this does (something like "prevent a panic if there's no user agent" iirc) in the final commit to master?

Comment (Joshua-Anderson) [inline settings/settings.go]: I'm not going to merge the first commit in this PR. I'll merge #123 and then rebase to clean up history.

Comment (arschles) [inline settings/settings.go]: got it. I was unclear that you were removing the first commit. carry on
